open-source-ecosystem/index.qmd

```diff
@@ -1,10 +1,14 @@
 ---
 title: VEDA Open Source Ecosystem
-subtitle: A modular ecosystem of open source software for science data platforms
+subtitle: Proliferating open standards for science data platforms
```

Comment:
I don't think you want `Proliferating` which implies that we're making many new standards all the time. We can discuss a better term, I assume you wanted a nicer word for Sharing/Creating?

Reply:
Ah, I've been using that word wrongly a lot, I see. 😅 

No, I meant "Increase the adoption of open standards" or "Promoting open standards"

Reply:
let's go with **Promoting** for now